packages/shared/src/types/multisig.ts

```diff
@@ -0,0 +1,9 @@
+import type { ChainId } from '@lifi/types';
+
+export interface MultisigProps {
+  destinationChain: ChainId;
+}
+export interface MultisigState extends MultisigProps {
+  // Destination Chain
+  onDestinationChainSelected: (chainId: number) => void;
```

Comment:
Maybe we should place these types closer to the store?

Reply:
you are correct, but I'm following the existing structure followed by Menu Store and settings store